### PR TITLE
Added missing translation files

### PR DESCRIPTION
**Pull request by tijsverkoyen**

### CODE REVIEW

File: translations/messages.nl.yaml [hunk 1]
```diff
@@ -0,0 +1,125 @@
+core:
+    interface:
+        backToTop: terug naar boven
+
+    menu:
+        home: home
+
+    warnings:
+        oldBrowser: <strong>Warning:</strong> You are using an older browser. If you want to see the site as it is intended, you may want to download a new version or a decent browser like <a href="http://www.google.com/chrome‎">Chrome</a>.
```
Comment (ohvitorino) [inline translations/messages.nl.yaml]: Still needs to be translated.

File: translations/messages.nl.yaml [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+datagrids:
+    actions:
+        edit: bewerk
+        show: toon
+
+datepicker:
+  pickDate: pick a date
```
Comment (ohvitorino) [inline translations/messages.nl.yaml]: idem

File: translations/messages.nl.yaml [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+  buttons:
```
Comment (ohvitorino) [inline translations/messages.nl.yaml]: idem

**Review by ohvitorino — CHANGES_REQUESTED** [review]

**Comment by tijsverkoyen** [comment]
@ohvitorino fixed in https://github.com/sumocoders/wireframes/pull/4/commits/ea3070f964ad08b3a9839d4c324f05217df50f56

**Review by ohvitorino — APPROVED** [review]

**Comment by ohvitorino** [comment]
👍

**Pull request merged**